docs/NOTICES.md

```diff
@@ -825,6 +826,18 @@
 ##### Affected files
 * [`routes.txt`](http://gtfs.org/reference/static#routestxt)
 
+#### [SameNameAndDescriptionForStopNotice](/RULES.md#SameNameAndDescriptionForStopNotice)
+##### Fields description
+
+| Field name     	| Description                             	| Type   	|
+|----------------	|-----------------------------------------	|--------	|
+| `csvRowNumber`   	| The row number of the faulty record.    	| String 	|
+| `stopId`        	| The id of the faulty record.            	| Long   	|
+| `routeDesc`    	| The faulty record's `stop_desc`.       	| String 	|
```

Comment:
typo: should be `stopDesc`

This is one more argument for removing this file. It is too easy to make a typo and it takes too much effort to catch all mistakes.

Reply:
Fixed in [PR #1004](https://github.com/MobilityData/gtfs-validator/pull/1004), opened [issue #1006](https://github.com/MobilityData/gtfs-validator/issues/1006) to discuss the issues we have with NOTICES.md